### DISCUSSION TITLE
Docker (python version)

### DISCUSSION THREAD

**Discussion by rodrigovimieiro** [CERN/TIGRE#606] [Q&A]
Have you tried to install TIGRE (python version) inside Docker?

Thanks!

I cannot compile it correctly

**Comment by AnderBiguri** [comment]
Nop never tried it... What is the issue? CUDA paths?

**Reply by rodrigovimieiro** [reply]
Exactly. 

Using this docker file:

```
FROM nvcr.io/nvidia/nvhpc:24.3-devel-cuda12.3-ubuntu22.04

ARG DEBIAN_FRONTEND=noninteractive

#setup layers
RUN apt-get update && apt-get install -y \
    libgdcm-dev \
    python3.6 \
    python3-pip \
    git \
    wget \
    pocl-opencl-icd \
    ocl-icd-dev \
    ocl-icd-opencl-dev \
    ocl-icd-libopencl1 \
    opencl-headers \
    freeglut3-dev \
    libgl1-mesa-glx \
    libgl1-mesa-dri \
    mesa-utils \
    xvfb \
    clinfo \
    && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

I also installed NVIDIA container-toolkit from this [link](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)

**Reply by AnderBiguri** [reply]
I wonder if https://github.com/CERN/TIGRE/issues/579 has to do with it. Admitedly this is super out of my field, I have barely ever used docker and I have no idea how its set up.

**Reply by rodrigovimieiro** [reply]
I don't think so. Ok, I will leave this open and I will update here if I have any advances
